Source/Core/VideoBackends/OGL/TextureCache.cpp

```diff
@@ -32,12 +33,24 @@ namespace OGL
 {
 static u32 s_ColorCbufid;
 static u32 s_DepthCbufid;
-static SHADER s_palette_pixel_shader[3];
+
+struct PaletteShader
+{
+  SHADER shader;
+  GLuint buffer_offset_uniform;
+  GLuint multiplier_uniform;
+  GLuint copy_position_uniform;
+};
+
+static PaletteShader s_palette_shader[3];
 static std::unique_ptr<StreamBuffer> s_palette_stream_buffer;
-static GLuint s_palette_resolv_texture;
-static GLuint s_palette_buffer_offset_uniform[3];
-static GLuint s_palette_multiplier_uniform[3];
-static GLuint s_palette_copy_position_uniform[3];
+static GLuint s_palette_resolv_texture = 0;
+
+static inline PaletteShader& RefPaletteShader(TLUTFormat tlutfmt)
```

Comment:
`inline` is very likely not necessary here. `static` already makes it a significant candidate for inlining since it's translation-unit-local.